packages/subgraph/schema.graphql

```diff
@@ -1514,63 +1505,20 @@ type ApprovalEvent implements Event @entity(immutable: true) {
     order: BigInt!
 
     """
-    The address that will be granting allowance to transfer ERC20/NFT.
+    The address that will be granting allowance to transfer ERC20.
     """
     owner: Account!
 
     """
-    The address that will be granted allowance to transfer ERC20/NFT.
+    The address that will be granted allowance to transfer ERC20.
     """
     to: Account!
 
-    """
-    Indicates whether the event was emitted for the approval of an NFT.
-    """
-    isNFTApproval: Boolean!
```

Comment:
We could arguably leave the NFT fields to reduce the chance of API usage breakage. Just mark them obsolete and have them be falsy/default values.

Reply:
we discussed this and decided to do a breaking change instead of leaving tech debt